packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -16,25 +16,23 @@ class TerraService {
   }
 
   /**
-   * Service hook executed before worker processes are launched.
+   * Service hook executed prior to initializing the webdriver session.
    * @param {object} config - The WebdriverIO configuration object.
    */
-  onPrepare(config) {
-    const { serviceOptions } = config;
+  beforeSession(config) {
+    global.Terra = {};
+    const { serviceOptions, ignoreComparisonResults } = config;
```

Comment:
where is `ignoreComparisonResults` coming from?

Reply:
`ignoreComparisonResults` used to be set as an ENV in wdio4. It is now being set as a CLI option: https://github.com/cerner/terra-toolkit/pull/545/files#diff-b821c3a083e050c54034354280dac343f0b8d9ad66c460cad80a99ec7ebea640

Reply:
Ahh yeah I had saw it in the CLI but didn't realize it was passed through to the service via the wdio CLI